quickjs.h

```diff
@@ -294,6 +294,7 @@ JS_EXTERN JSRuntime *JS_NewRuntime(void);
 /* info lifetime must exceed that of rt */
 JS_EXTERN void JS_SetRuntimeInfo(JSRuntime *rt, const char *info);
 JS_EXTERN void JS_SetMemoryLimit(JSRuntime *rt, size_t limit);
+JS_EXTERN void JS_SetDumpFlags(JSRuntime *rt, unsigned flags);
```

Comment:
```suggestion
JS_EXTERN void JS_SetDumpFlags(JSRuntime *rt, uint32_t flags);
```

Since that's how we define the field internally? I'd also make it uint64_t, since we might add more in the future...

Reply:
The current implementation uses 24 bits, but we can make the API more futureproof.

Reply:
Done with `uint64_t flags`